docs/ml/upload-model.md

```diff
@@ -8,25 +8,60 @@ aliases:
   - /manage/data/upload-model/
   - /manage/ml/upload-model/
 description: "Upload a Machine Learning model to Viam to use it with the ML Model service."
-# SME: Aaron Casas
+# SME: Steven B.
 ---
 
-On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **Models** subtab.
+Upload a new model to your organization with the option to share it in the [Viam registry](https://app.viam.com/registry).
+Or, upload a new version of a published model.
 
-![Add new model](/ml/add-new-model.png)
+Also, [share a model from your organization](#make-an-existing-model-public-in-the-registry) with other users in the registry.
 
-To add a new model:
+## Upload a new model or new version
 
-1. Specify a **Name** for the model.
-2. Add a `.tflite` model file.
-3. Add a `.txt` label file.
-4. Click **CREATE MODEL**.
+On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, navigate to the **MODELS** subtab.
+Click **Upload model**.
 
-The model is now added and becomes visible in the **Models** section of the page.
+{{<imgproc src="/ml/add-new-model.png" resize="400x" alt="Upload model menu on the DATA tab of the Viam app.">}}
 
-![The trained model](/ml/stars-model.png)
+Select if you want to upload a **New model** or a **New version** of a previously published model.
 
-### Upload a new version of a model
+{{< tabs >}}
+{{% tab name="New model" %}}
+
+Select **Public** to upload the model publicly to the [Viam registry](https://app.viam.com/registry).
+This makes the model usable by any external organization.
+See [ML models in the registry](/registry/#ml-models) for more information.
+
+Select **Private** to only publish the model for internal use within this {{< glossary_tooltip term_id="organization" text="organization" >}}.
+Select **Next steps** to continue.
+
+{{<imgproc src="/ml/upload-model.png" resize="600x" alt="Upload model menu on the DATA tab of the Viam app.">}}
+
+1. Specify a name for the model.
+2. Specify the **Model type**.
+3. Add a `.tflite` model file.
+4. Add a `.txt` label file.
+5. Add a short description.
+   This will help other users to understand how to use your model.
+
+Select **Upload model**.
+The model is now visible on the **MODELS** subtab of the **DATA** tab in the [Viam app](https://app.viam.com) and available to [deploy on your machine](/ml/deploy/).
+Also, if you selected **Public**, it should be publicly visible in the [Viam registry](https://app.viam.com/registry).
+
+{{% /tab %}}
+{{% tab name="New version" %}}
+
+{{<imgproc src="/ml/select-existing-model.png" resize="400x" alt="Select from your existing models.">}}
+
+1. Select the model you would like to update from your existing models.
+2. Select **Next steps** to continue.
+3. Add an updated `.tflite` model file.
+4. Add an updated `.txt` label file.
+5. Click **Upload model**.
+
+Your model is now updated.
+
+{{% alert title="Note" color="note" %}}
```

Comment:
I think this note may need checking. I think I saw yesterday that the **COPY** button is no longer there and it's now a slightly different UI? Can you double check?

Reply:
You are correct about that, updated!